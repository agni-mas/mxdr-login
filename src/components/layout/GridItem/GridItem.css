.item {
    border: 1px solid gray;
    padding: 30px;
    border-radius: 5px;
    height: min-content;
}

.grid-item-title {
    font-weight: 200;
    font-family: 'Graphik';
    font-size: 22px;
    color: white;
    margin-bottom: 20px;
}