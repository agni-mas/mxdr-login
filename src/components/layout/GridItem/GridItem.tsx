import React, { FC } from 'react';
import './GridItem.css'

interface DashboardProps {
  children: any,
  title: string,
}

const Dashboard: FC<DashboardProps> = ({ children, title }) => (
  <section className='item'>
    <h2 className='grid-item-title'>{title}</h2>
    {children}
  </section>
);

export default Dashboard;
