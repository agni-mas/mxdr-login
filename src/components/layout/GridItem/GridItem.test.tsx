import { render } from "@testing-library/react";
import GridItem from "./GridItem";

it("renders <GridItem/>", () => {
    const props = {
        children: <div>Some Chart Component</div>,
        title: 'Hello world'
    }
    const { asFragment } = render(<GridItem children={props.children} title={props.title} />);
    expect(asFragment()).toMatchSnapshot();
});