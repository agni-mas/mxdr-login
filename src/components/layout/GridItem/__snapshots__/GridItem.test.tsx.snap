// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <GridItem/> 1`] = `
<DocumentFragment>
  <section
    class="item"
  >
    <h2
      class="grid-item-title"
    >
      Hello world
    </h2>
    <div>
      Some Chart Component
    </div>
  </section>
</DocumentFragment>
`;
